packages/teams-js/src/private/externalAppAuthentication.ts

```diff
@@ -498,6 +498,59 @@ export namespace externalAppAuthentication {
     });
   }
 
+  /**
+   * @beta
+   * @hidden
+   * API to authenticate power platform connector plugins
+   * @internal
+   * Limited to Microsoft-internal use
+   * @param titleId ID of the acquisition
+   * @param pluginId pluginId of the connector plugin
+   * @param signInUrl signInUrl for the connctor page listing the connector. This is optional
+   * @param oauthWindowParameters parameters for the signIn window
+   * @returns A promise that resolves when authentication succeeds and rejects with InvokeError on failure
+   */
+  export function authenticateWithPowerPlatformConnectorPlugins(
+    titleId: string,
+    pluginId?: string,
+    signInUrl?: string,
+    oauthWindowParameters?: OauthWindowProperties,
+  ): Promise<void> {
+    ensureInitialized(runtime, FrameContexts.content);
+
+    if (!isSupported()) {
+      throw errorNotSupportedOnPlatform;
+    }
+
+    validateId(titleId, new Error('titleId is Invalid.'));
+    if (pluginId) {
+      validateId(pluginId, new Error('pluginId is Invalid.'));
+    }
+    if (signInUrl) {
+      validateUrl(signInUrl);
+    }
+
+    return sendMessageToParentAsync(
+      getApiVersionTag(
+        externalAppAuthenticationTelemetryVersionNumber,
+        ApiName.ExternalAppAuthentication_AuthenticateWithOauth2,
```

Comment:
Api name needs an update to `authenticateWithPowerPlatformConnectorPlugins`, and also needed to add in ApiName in Telemetry.ts